package.json

```diff
@@ -9,7 +9,7 @@
     "change": "beachball change",
     "clean": "lage clean -- --color",
     "format": "format-files -i -style=file -assume-filename=../.clang-format",
-    "format:verify": "format-files -i -style=file -assume-filename=../.clang-format -verify",
```

Comment:
`-assume-filename=../.clang-format` was being used incorrectly. It's used to override language auto-detection.

> -assume-filename= option can be used to override the file name clang-format uses to detect the language.